test/test-pages/mozilla-1/expected.html

```diff
@@ -7,14 +7,7 @@
                 <div class="animation-wrapper" id="flexible-top-animation"><img src="http://mozorg.cdn.mozilla.net/media/img/firefox/desktop/customize/animations/flexible-top-fallback.c960365ba781.png" class="fallback" alt=""/></div>
             </div>
         </section>
-        <section id="designed" class="ga-section" data-ga-label="Designed to be redesigned">
-            <div class="container">
-                <div id="designed-copy">
-                    <h2>Designed to <br class="wide-br"/>be redesigned</h2>
-                    <p>Get fast and easy access to the features you use most in the new menu. Open the “Customize” panel to add, move or remove any button you want. Keep your favorite features — add-ons, private browsing, Sync and more — one quick click away.</p><img class="js " src="" data-processed="false" data-src="//mozorg.cdn.mozilla.net/media/img/firefox/desktop/customize/designed-redesigned.fbd3ee9402e6.png" data-high-res="true" data-high-res-src="//mozorg.cdn.mozilla.net/media/img/firefox/desktop/customize/designed-redesigned-high-res.6efd60766484.png" alt="" id="designed-mobile"/></div>
-                <div class="animation-wrapper" id="flexible-bottom-animation"><img src="http://mozorg.cdn.mozilla.net/media/img/firefox/desktop/customize/animations/flexible-bottom-fallback.cafd48a3d0a4.png" class="fallback" alt=""/></div>
-            </div>
```

Comment:
No, it's not really acceptable. I don't know off-hand why this is getting removed... You could add logging and tweak the debug flag to get the logging to show up and then only run this test (`mocha test/test-readability.js -g mozilla`) or the testcase generation script, to see why it does this thing.

Oh, maybe I do know.... before this change, was all the whitespace between nodes being converted to `<p>` tags which then added weighting to the container which led to it being kept, given that it also has images and we have some logic about removing image containers without too much text and so on?